src/mockData/facebookAuth.js

```diff
@@ -0,0 +1,4 @@
+const user = {
+  id: '1985674401577712', method: 'facebook', firstName: 'Ishimwe', lastName: 'William', email: 'ishimwewilliam@yahoo.fr',
```

Comment:
put each key-value pair on a new line. apply it in all `mockData` files.

Reply:
done, thanks!